include/usearch/index.hpp

```diff
@@ -3272,7 +3449,6 @@ class index_gt {
 
         // We are loading an empty index, no more work to do
         if (!header.size) {
-            reset();
```

Comment:
This is not related to this PR but I add this PR because I need similar change in `view()`. If this should be a separated PR, I'll open a separated PR.

We don't need to call `reset()` here because we don't change anything after the above `reset()`.